Assets/Scripts/ArrowReflector.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class ArrowReflector : MonoBehaviour
+{
+    /*
+     * Simple script that 'refelcts' an object after it collides with the surface.
+     * Controllable through an EnergyMultiplyer to change the force, and a BoostHeight to add a value to the Y velocity
+     */
+
+    //a multiplyer that affects the resulting velocity
+    //  negitive values are more realistic while positive add energy
+    [SerializeField]
+    private float EnergyMultiplyer = -.9f;
+
+    //a boost to hight to correct for drop when reaching the target, adds directly to the Y of the velocity
+    [SerializeField]
+    private float BoostHeight = 0f;
+
+    private void OnCollisionEnter(Collision collision)
+    {
+        //defines varibles to edit collision object
+        var rigidBody = collision.rigidbody;
+        var velocity = rigidBody.velocity;
+
+        //adds the velocity and normalized venocity to get a good approxamation of the reflection angle,
+        //then multiplies by the velocites magnatude and energy multipyer to get resulting force
+        var result = (rigidBody.velocity.normalized - collision.GetContact(0).normal) * velocity.magnitude * EnergyMultiplyer;
```

Comment:
use Vector3.Reflect instead. I don't think you can get a reflection vector without using trig or dot products

Reply:
got the reflet function working with a good method that I came up with
![Screenshot 2020-10-14 005508](https://user-images.githubusercontent.com/71116099/95960005-13aa3f00-0db8-11eb-89bd-ede578f56992.png)
now I'm working on translating it so it follows the reflection angle better

Reply:
It uses a trigger that gets the velocity of the arrow as it comes near, so the arrow doesn't have to be constantly storing its own velocity

Reply:
![result](https://user-images.githubusercontent.com/71116099/96029275-4a5f7400-0e0f-11eb-83cd-568b1c21fc6f.png)
100% complete, arrow reflects and is transformed along the new path